changelogs/fragments/sanity_test_ignore_file.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
```

Comment:
```suggestion
trivial:
```

Reply:
@justjais Thank you for the review, I have fixed the changelog according to your suggestion.